internal/pkg/util/fs/overlay/overlay_linux.go

```diff
@@ -31,7 +31,7 @@ const (
 	upperDir
 )
 
-type fs struct {
+type filesys struct {
```

Comment:
Please don't change unless necessary, as I don't think it's used here.

If it is necessary to change it, keep it consistent. `incompatibleFs` should now be `incompatibleFilesys`.

Reply:
Several tests in internal/pkg/util/fs/overlay/overlay_item_linux_test.go and internal/pkg/util/fs/overlay/overlay_set_linux_test.go now use fs.CopyFile() from internal/pkg/util/fs/helper.go. While in principle one could have imported "github.com/sylabs/singularity/internal/pkg/util/fs" as <some other name>, in practice the fs.CopyFile() usage is widespread in our code, and I think somebody seeing something like filesys.CopyFile() in only this place would get the impression that this was something other than the ordinary fs.CopyFile() usage.

I'll go ahead and change `incompatibleFilesys` accordingly.